python/cuda_parallel/cuda/parallel/experimental/__init__.py

```diff
@@ -1,3 +1,12 @@
 # Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
 #
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import logging
+
+from cuda.bindings.path_finder import (  # type: ignore[import-not-found]
+    _load_nvidia_dynamic_library,
+)
+
+for libname in ("cudart", "nvrtc", "nvJitLink"):
```

Comment:
ditto, I suppose cudart is not needed anymore?

Reply:
Yes, thanks, I missed this before, done: commit ce889d5c419bd8330b2c58e5363f92c118b7df03